views/partials/beerpartial.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="beer-card">
+    <div class="image-container"><img src="{{this.image_url}}" alt="beer-photo"></div>
```

Comment:
Since not all beers have images, I would recommend to wrap this line of code in an if statement so that the image only display it there is one. Currently it's displaying an image icon and the alternative text.